e2e/browser/test/fixtures.ts

```diff
@@ -83,23 +84,48 @@ export type TestContainer = {
   privateFileText: string;
 };
 
+export async function retryAsync<T>(
+  callback: () => Promise<T>,
+  maxRetries = 5,
+  interval = 5_000,
+): Promise<T> {
+  let tries = 0;
+  while (tries < maxRetries) {
+    try {
+      // The purpose here is to retry an async operation, not to parallelize.
+      // Awaiting the callback will throw on error before returning.
+      // eslint-disable-next-line no-await-in-loop
+      return await callback();
+    } catch (e: unknown) {
+      tries += 1;
+      // eslint-disable-next-line no-await-in-loop
+      await new Promise((resolve) => {
+        setTimeout(resolve, interval);
+      });
+    }
+  }
+  throw new Error(
+    `An async callback is still failing after ${maxRetries} retries.`,
```

Comment:
Can you also include the 5 error messages in this so that we can see what goes wrong when it fails after 5 retries.

Reply:
Good idea: https://github.com/inrupt/solid-client-authn-js/pull/3105/commits/ddb4d57c65aef58d25bb64594080f328cede001e